### ISSUE TITLE
Error when piping output into a file

### ISSUE THREAD

**Issue by pointydev** [adam10603/ChatScraper#1]
```
[pointy@pointy ChatScraper]$ node scrape_chat --dict="You are unable to participate in this channel" --vods-from=psp1g --max-vods=10 --print-links --plain > pspbanned.txt
/home/pointy/tools/ChatScraper/scrape_chat.js:376
            process.stdout.clearLine();
                           ^

TypeError: process.stdout.clearLine is not a function
    at ChatDownloader.<anonymous> (/home/pointy/tools/ChatScraper/scrape_chat.js:376:28)
    at ChatDownloader.emit (node:events:513:28)
    at retrieveData (/home/pointy/tools/ChatScraper/utils/twitch/chat_downloader.js:166:38)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.all (index 0)
    at async ChatDownloader.getChatReplays (/home/pointy/tools/ChatScraper/utils/twitch/chat_downloader.js:172:14)

Node.js v18.16.0
```

This error does not occur when outputting to the terminal.

Thanks,
pointy

**Comment by adam10603** [comment]
i didnt even realize people can post issues, i never check these 💀 good thing i just noticed.
ill fix this in a bit.

**Comment by adam10603** [comment]
fixed in v1.1.1
